erpnext/assets/doctype/asset_repair/asset_repair.py

```diff
@@ -248,8 +248,8 @@ def validate_serial_no(self, stock_item):
 			)
 
 	def increase_stock_quantity(self):
```

Comment:
This function is no longer required, the framework will throw a validation on cancellation of asset repair, asking to cancel the stock entry first. Let the user cancel the stock entry manually, it is not a good idea to ignore the link while cancelling.